pkg/scalers/externalscaler/externalscaler.proto

```diff
@@ -40,5 +43,9 @@ message GetMetricsResponse {
 
 message MetricValue {
     string metricName = 1;
+
+    // deprecated, use metricValueFloat instead
     int64 metricValue = 2;
+
+    double metricValueFloat = 3;
```

Comment:
This actually breaks the GRPC contract as I explained here -> https://github.com/kedacore/keda/pull/6482#pullrequestreview-2559038091

Currently will work just because the server has been updated too, but for other proto files this breaks. Make all the new fields `optional` please.

Reply:
@JorTurFer all fields in proto definition are optional (imho)

Reply:
Yeah, agree with @jkremser here

Reply:
it uses `optional` in front
```suggestion
    optional double metricValueFloat = 3;
```

ref: https://protobuf.dev/programming-guides/proto3/#field-labels